generateData.js

```diff
@@ -2,7 +2,6 @@ const fs = require('fs');
 const csvWriter = require('csv-write-stream');
 let writer = csvWriter();
 const faker = require('faker');
-// let counter = 0;
 
 const createTags = () => {
```

Comment:
Took a minute to understand what was going on with this function but I like your logic here.

Reply:
@codysmith287 thanks for letting me know! I went back and added a comment to explain the function and also renamed some of the variables.